src/crs/geographic.jl

```diff
@@ -339,3 +339,8 @@ Base.convert(::Type{LatLon}, coords::Cartesian{Datum,3}) where {Datum} = convert
 
 Base.convert(::Type{Cartesian}, coords::LatLonAlt{Datum}) where {Datum} = convert(Cartesian{Datum}, coords)
 Base.convert(::Type{LatLonAlt}, coords::Cartesian{Datum,3}) where {Datum} = convert(LatLonAlt{Datum}, coords)
+
+Base.convert(::Type{LatLon}, coords::LatLonAlt{Datum}) where {Datum} = convert(LatLon{Datum}, coords)
+Base.convert(::Type{LatLon{Datum}}, coords::LatLonAlt{Datum}) where {Datum} = LatLon{Datum}(coords.lat, coords.lon)
+Base.convert(::Type{LatLonAlt}, coords::LatLon{Datum}) where {Datum} = convert(LatLonAlt{Datum}, coords)
+Base.convert(::Type{LatLonAlt{Datum}}, coords::LatLon{Datum, Deg{T}}) where {Datum, T} = LatLonAlt{Datum}(coords.lat, coords.lon, zero(Met{T}))
```

Comment:
We are not using `Deg{T}` in this dispatch, there is probably a better way to do this with the other fields. @eliascarv appreciate if you can take a look at it.

Reply:
Using `numtype(coord.lon)` as suggested by @eliascarv now.